nipype/algorithms/compcor.py

```diff
@@ -22,6 +22,15 @@ class CompCoreOutputSpec(TraitedSpec):
 class CompCore(BaseInterface):
     '''
     Node with core CompCor computation, used in aCompCor and tCompCor
+
+    Example
+    -------
+
+    >>> ccinterface = CompCore()
```

Comment:
yes, I tried that earlier. It didn't work, I tried for ~10 minutes to get it to work, then decided it wasn't really worth it for now. Should I consider it worth it for now?